console/console-init/ui/src/modules/iot-device/components/CreateMetadata/MetaDataRow.tsx

```diff
@@ -77,21 +108,16 @@ export const MetaDataRow: React.FC<IMetaDataRow> = ({
     return validationStatus;
   };
 
-  //TODO: Call graphql queries and populate options in SelectOption
-  const handlePropertyInputChange = async (propertyKey: string) => {
-    let updatedPropertyMetadata = [...metadataList];
-    updatedPropertyMetadata[rowIndex].key = propertyKey;
-    setMetadataList(updatedPropertyMetadata);
-  };
-
-  const handleAddChildRow = (event: any) => {
-    let parentKey: string = metadataRow.key;
-    let newRow = {
+  const handleAddChildRow = (_: any) => {
```

Comment:
We can omit the param from here as well.

Reply:
Agreed